netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1275,13 +1239,70 @@ _net_ebpf_ext_process_redirect_verdict(
     NET_EBPF_EXT_RETURN_NTSTATUS(status);
 }
 
-/*
- * Default action is BLOCK. If this callout is being invoked, it means at least one
- * eBPF program is attached. Hence no connection should be allowed unless allowed by
- * the eBPF program.
+/**
+ * @brief This function determines if the sock_addr eBPF program should be invoked as part of processing the classify
+ * callback at CONNECT_REDIRECT layer.
+ *
+ * @param[in] filter_context Pointer to net_ebpf_extension_sock_addr_wfp_filter_context_t associated with WFP filter.
+ * @param[in] sock_addr_ctx Pointer to bpf_sock_addr_t struct to be passed to eBPF program.
+ * @param[in] v4_mapped Boolean indicating if the IP address in sock_addr is v4 mapped v6 address or not.
  *
- * If the eBPF program verdict is BLOCK, no context is created and AUTH classify will
- * block the connection by default.
+ * @returns True if eBPF program should be invoked, False otherwise.
+ */
+
+bool
```

Comment:
Missing ```_Must_inspect_result_``` annotation? IMO as a general rule, if a function returns _any_ sort of a result, we _must_ use the 'must check' annotation.